src/main/java/uk/gov/hmcts/reform/pip/account/management/validation/validator/AzureEmailValidator.java

```diff
@@ -10,6 +10,7 @@ public class AzureEmailValidator implements ConstraintValidator<AzureEmail, Stri
 
     @Override
     public void initialize(AzureEmail email) {
+        // Do nothing
```

Comment:
The method "initialize" in the parent interface is a default method. It is optional whether we override that or not. If nothing to override, we can just remove this method from here.